### ISSUE TITLE
Measure (and sum up) web content's memory too

### ISSUE THREAD

**Issue by xabolcs** [mozilla/memchaser#231]
With e10 enabled, `Resident` usage dropped to ~600 MB from ~1300 MB in my case.

It would be nice to show the web content's (e.g. `resident`) size too at least as a separate measure, near the `Resident` label.
It would be even nice to include a preference to display them as one summarized memory usage.

**Comment by whimboo** [comment]
Maybe we differentiate between `Chrome` and `Content` in the toolbar.

**Comment by xabolcs** [comment]
@amccreight, it is possible right now for an addon to check <del>these</del> web content's `resident` easily, <del>but</del> or it <del>should</del> would be easy to add something to Firefox to allow that?

Doing a memory report at every five seconds is not acceptable. ;)

**Comment by amccreight** [comment]
Yes, I think the residentFast field on nsIMemoryReporter doesn't do a memory report so that should be fine. I think most of those fields don't do a full report.

http://mxr.mozilla.org/mozilla-central/source/xpcom/base/nsIMemoryReporter.idl#385
